Makefile

```diff
@@ -199,6 +199,13 @@ githooks-commit:
 githooks-push:
 	@if [ ! -e .git/hooks/pre-push ]; then ln -s ../../bin/pre-push .git/hooks/pre-push; fi
 
+list-publishable-modules:
+	./bin/list-publishable-modules.js
+
+# publishes module given via path to NPM: make publish-module path=PATH_TO_MODULE
+publish-module:
+	make -C $(path) publish
```

Comment:
Sorry I didn't notice this before, but if `make publish-module path=PATH_TO_MODULE` is the way to use this, then please change the name of the variable `path` to something less generic (especially since it's easy to confuse it with the almighty `PATH` environment variable).

The question is - do we actually need this rule in the main Makefile? Maybe just make the `list-publishable-modules.js` script print the list of the publishable modules along with the `make` command to publish each of them? Because to me `make publish-module path=PATH_TO_MODULE` isn't necessarily easier to remember and/or type than `make -C PATH_TO_MODULE publish` :grin: